<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WScarpiera</class>
 <widget class="QWidget" name="WScarpiera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="scafMBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>scafL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allObjsBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>All Objects</string>
   </property>
  </widget>
  <widget class="QPushButton" name="antaLBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>AntaL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafHBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtrn">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to OpenSpace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="antaHBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>AntaH</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
